daemon/src/model/cfd.rs

```diff
@@ -545,7 +549,7 @@ impl Cfd {
         self.collaborative_settlement_finality || self.cet_finality || self.refund_finality
     }
 
-    pub fn start_contract_setup(&self) -> Result<(SetupParams, Identity)> {
+    pub fn start_contract_setup(&self) -> Result<(Event, SetupParams, Identity)> {
```

Comment:
I think we can move `SetupParams` and `Identity` into the `ContractSetupStarted` event. It might be useful for the UI for example to display the params of the setup.

Reply:
It will also simplify this return type and make it uniform with the other handlers!

Reply:
I might do it later, if UI requires it - the price for changing it now in making it really awkward to retrieve the setup params at the callsite (helper with an unwrap :/ due to downcasting to the particular event)

Reply:
Yeah I saw it on Lucas' branch. That is quite unfortunate ...

Happy to leave as is for now.

Reply:
I've actually got a follow-up mostly ready, but without the Accepted event it's actually a regression in functionality as per this patch, so I'll save it up for the follow-up.